lib/ansible/plugins/action/package.py

```diff
@@ -29,6 +29,9 @@ class ActionModule(ActionBase):
 
     TRANSFERS_FILES = False
 
+    BUILTIN_PKG_MGR_MODULES = set(['apk', 'apt', 'dnf', 'homebrew', 'installp', 'macports', 'opkg', 'portage', 'pacman',
+                                   'pkg5', 'pkgin', 'pkgng', 'sorcery', 'svr4pkg', 'swdepot', 'swupd', 'urpmi', 'xbps', 'yum', 'zypper'])
+
```

Comment:
don't hardcode these lists again, we already have a hardcoded list in facts, use that